src/compiler/generate-common-tests.xsl

```diff
@@ -421,6 +421,7 @@
             <xsl:when test="x:apply">
                <xsl:variable name="local-params" as="element(x:param)*" select="x:apply/x:param"/>
                <xsl:element name="{x:xspec-name('apply')}" namespace="{$xspec-namespace}">
+                  <xsl:apply-templates select="x:apply" mode="x:copy-namespaces" /><!--TODO: Check that this line works whenever x:apply is implemented.-->
                   <xsl:sequence select="$apply/@*"/>
```

Comment:
Is it possible to replace this
```xml
<xsl:element name="{x:xspec-name('apply')}" namespace="{$xspec-namespace}">
  <xsl:apply-templates select="x:apply" mode="x:copy-namespaces" />
```
with this?

```xml
<xsl:for-each select="x:apply">
  <xsl:copy>
```

And once we migrate to XSLT 3.0, just
```xml
<xsl:copy select="x:apply">
```

Reply:
Also [x:context](https://github.com/xspec/xspec/pull/640/files#diff-001d11aaadf3edf13d9a7c6cfc30b845R444) and [x:call](https://github.com/xspec/xspec/pull/640/files#diff-001d11aaadf3edf13d9a7c6cfc30b845R468).

Reply:
Yes, I switched to the for-each/copy approach in those 3 spots, in commit 0b34eb3190493f3da9e29d459454396f74a86d82